### DISCUSSION TITLE
Someone explain this chart what is saying

### DISCUSSION THREAD

**Discussion by Chiyan200** [openai/whisper#2216] [Q&A]
1. I can't understand this chart what is WER and CER 
2. If any language has Higher rate perform well or not 

![chart.png](https://github.com/openai/whisper/assets/94426967/b2ab0bca-f00f-4964-9dd5-12ec0ba1de35)

**Comment by glangford** [comment]
The README describes the chart and the meaning of WER and CER. 

See https://github.com/openai/whisper?tab=readme-ov-file#available-models-and-languages

> The figure below shows a performance breakdown of large-v3 and large-v2 models by language, using WERs (word error rates) or CER (character error rates, shown in Italic) evaluated on the Common Voice 15 and Fleurs datasets

It describes the figure as a "performance breakdown" but it is really about language accuracy for the given dataset. The longer the bar, the more errors for that language.

**Reply by Chiyan200** [reply]
Thank you so much
